tests/zipfile/test_unzip.py

```diff
@@ -7,6 +7,15 @@
 from cookiecutter.exceptions import InvalidZipRepository
 
 
+@pytest.fixture
+def clone_dir(tmp_path):
```

Comment:
hey @SharpEdgeMarshall 👋 , 

what do you think about moving this fixture in the `conftest.py` file to avoid fixtures duplication?

Thanks 🙏